pico/api/src/main/java/io/helidon/pico/api/ServiceProviderBindable.java

```diff
@@ -108,11 +108,20 @@ default void rootProvider(ServiceProvider<T> rootProvider) {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the previously assigned {@link PicoServices} instance.
+     *
+     * @return the previously assigned pico services instance, or empty if never assigned
+     * @see #picoServices(Optional)
+     */
+    Optional<PicoServices> safePicoServices();
```

Comment:
Should be called `picoServices`, this name is strange.

Reply:
There is a conflict with `picoServices` on AbstractProvider - that method should be removed, and this one used instead (with `orElseThrow()` maybe)

Reply:
Addressed and pushed.